Mail/Mailer.cs

```diff
@@ -0,0 +1,124 @@
+using System;
+using System.IO;
+using System.Text;
+using MailKit.Net.Smtp;
+using MimeKit;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+
+namespace NetCoreBootstrap.Mail
+{
+    public class Mailer
+    {
+        private readonly SmtpClient _smtpClient;
+        private string host, username, password, name, email;
+        private int hostPort;
+        private readonly string _jsonFilePath;
+        private readonly char _jsonSplitter = ':';
+        
+        public Mailer()
+        {
+            _smtpClient = new SmtpClient();
+            _jsonFilePath = "appsettings.Development.json";
+            SetAccountConfiguration();
+        }
+
+        public void Send(string toName, string toAddress, string subject, string body, string type = "plain")
+        {
+            var message = new MimeMessage { Subject = subject, Body = new TextPart (type) { Text = body } };
```

Comment:
use like this:
var message = new MimeMessage { Subject = subject, Body = new TextPart(type) { Text = body }};